app/views/doc/ref.html.erb

```diff
@@ -7,9 +7,10 @@
   <div class='callout quickref'>
     <p>
       Quick reference guides:
-      <a href="https://github.github.com/training-kit/downloads/github-git-cheat-sheet.pdf">GitHub Cheat Sheet</a>
-      <small class='light'>(PDF) &nbsp;|&nbsp;</small>
-      <a href="http://ndpsoftware.com/git-cheatsheet.html">Visual Git Cheat Sheet</a>
+      <%= link_to "GitHub Cheat Sheet", "https://github.github.com/training-kit/downloads/github-git-cheat-sheet.pdf" %>
+      <small class='light'>(PDF)</small>
+      |
```

Comment:
small note: now the pipe (`|`) is not under the css `.light` class. Is this intended?

Reply:
see difference: https://git-scm-pr-1389.herokuapp.com/docs vs https://git-scm.com/docs

Reply:
Yes, this is intentional. 
The light text is only used for secondary information (hinting at the file type), but the pipe separates the two primary entries, therefore imho it shouldn't be light.

It was part of the change in #1388.
See here: https://git-scm.com/doc